### DISCUSSION TITLE
what is this on 1.2.8?

### DISCUSSION THREAD

**Discussion by xmz0916** [moononournation/Arduino_GFX#201] [Q&A]
ESP32_2432S028R
ESP32_3248S035R
ESP32_4827S043

**Comment by moononournation** [comment]
![image](https://user-images.githubusercontent.com/13228891/189345665-8f9d696f-d432-4127-b714-c16b7dd2fe77.jpeg)

**Reply by moononournation** [reply]
![image](https://user-images.githubusercontent.com/13228891/189345716-98798010-e85b-4dfc-b0ae-9279c2bb818e.jpeg)

**Reply by moononournation** [reply]
https://www.aliexpress.com/item/1005004632953455.html?spm=5261.ProductManageOnline.0.0.67744edfeS57Kj

https://www.aliexpress.com/item/1005004502250619.html?spm=5261.ProductManageOnline.0.0.67744edfeS57Kj
